runtime/vm/visible.c

```diff
@@ -29,6 +29,62 @@
 #include "util_api.h"
 #include "vm_internal.h"
 
+
+
+/**
+ * Check module access from srcModule to destModule
```

Comment:
Can you expand on the checks that occur?  The reflective vs not difference,  unnamed modules, etc?  A good comment here can save significant debugging time later.